Makefile

```diff
@@ -50,8 +55,10 @@ TEST_TAG_FLAG := -tags $(ALL_TEST_TAGS)
 
 ##### Variables ######
 
-GOBIN := $(if $(shell go env GOBIN),$(shell go env GOBIN),$(GOPATH)/bin)
-PATH := $(GOBIN):$(PATH)
+LOCALBIN := .bin
+STAMPDIR := .stamp
+PATH := $(LOCALBIN):$(PATH)
+GOINSTALL := GOBIN=$(shell pwd)/$(LOCALBIN) go install
```

Comment:
shouldn't this use $(ROOT)?

Reply:
It should, good catch. I added $(ROOT) later

Reply:
actually wait, why do we need ROOT? I thought it was for invoking make if the current dir is not the root of the repo, but invoking make doesn't work in that case anyway

Reply:
root isn't used for invoking make but for invoking other commands from subdirectories